pgml-dashboard/app/static/css/base.css

```diff
@@ -602,6 +602,10 @@ main turbo-frame:first-of-type .notebook-cell {
   margin-bottom: 2px;
 }
 
+.notebook-cell.selected {
+    border: 2px solid green;
```

Comment:
Not a fan of plain green, but we can leave it for the designer.

A very light gray shading on cell hover background might also help to scope the button actions visually.

Reply:
We need to think of accessibility. Light colors are barely visible.